get_s3_test.go

```diff
@@ -9,17 +9,8 @@ import (
 	"github.com/aws/aws-sdk-go/aws/awserr"
 )
 
-func init() {
-	// These are well known restricted IAM keys to a HashiCorp-managed bucket
-	// in a private AWS account that only has access to the open source test
-	// resources.
-	//
-	// We do the string concat below to avoid AWS autodetection of a key. This
-	// key is locked down an IAM policy that is read-only so we're purposely
-	// exposing it.
-	os.Setenv("AWS_ACCESS_KEY", "AKIAITTDR"+"WY2STXOZE2A")
-	os.Setenv("AWS_SECRET_KEY", "oMwSyqdass2kPF"+"/7ORZA9dlb/iegz+89B0Cy01Ea")
-}
+//Note for external contributors: In order to run the GCS test suite, you will only be able to be run
```

Comment:
Small formatting nit and typo fix.
```suggestion
// Note for external contributors: In order to run the S3 test suite, you will only be able to be run
```

Reply:
🤦🏻‍♀️ this is why i need more coffee! thank you

Reply:
😄 ☕